src/main/java/org/folio/circulation/domain/UpdateRequestQueue.java

```diff
@@ -137,6 +136,20 @@ public static UpdateRequestQueue using(Clients clients) {
     }
   }
 
+  CompletableFuture<Result<RequestAndRelatedRecords>> onMoved(
+    RequestAndRelatedRecords requestAndRelatedRecords) {
+    final Request request = requestAndRelatedRecords.getRequest();
+    final RequestQueue requestQueue = requestAndRelatedRecords.getRequestQueue();
+    if (requestAndRelatedRecords.getOriginalItemId().contentEquals(request.getItemId())) {
```

Comment:
Did you really want `contentEquals` here? I believe both are `String` objects, so I would think `equals` would be more appropriate here.

Reply:
Good find. I will fix.